src/Model.ts

```diff
@@ -346,6 +348,31 @@ export class Model implements IModel {
   }
 
   /**
+   * Compares a regular object to a model.
+   *
+   * @param {object} comparingObject Object that will be compared to the model
+   * @param {IModelIsEqualParams} params Options for comparison
+   * @returns {boolean} Is the object equal to the model
+   * @memberof Model
+   */
+  public isEqual(comparingObject: object, params?: IModelIsEqualParams): boolean {
+    if (!params) {
```

Comment:
1. Don't change arguments - nothing good can happen...
2. Why not use default values and set it as required (not nullable)?